rotkehlchen/tests/unit/test_tasks_manager.py

```diff
@@ -667,3 +668,5 @@ def test_maybe_update_aave_v3_underlying_assets(
         assert task_manager.database.get_static_cache(
             cursor=cursor, name=DBCacheStatic.LAST_AAVE_V3_ASSETS_UPDATE,
         ) is not None
+
+    assert task_manager.database.msg_aggregator.rotki_notifier.pop_message() is None  # type: ignore[union-attr]  # rotki_notifier is MockRotkiNotifier
```

Comment:
there is a `num_messages` or something similar that you can use and assert it is 0. I believe that is clearer